contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -987,8 +985,7 @@ GRANT SELECT ON sys.procedures TO PUBLIC;
 
 create or replace view sys.sysforeignkeys as
 select
-  c.conname as name
-  , c.oid as object_id
+  c.oid as constid
```

Comment:
All the datatypes in this view are inconsistent with T-SQL. I will suggest you to check all the system views for datatype inconsistency and prepare notes of that, it will also help us reviewing the PR.